engine/tests/test_engine.bats

```diff
@@ -35,274 +31,267 @@ test_vtd_plugin_exists() {
     # The plugin section and the registry section have path references, which
     # are not reproducible, so we need to rewrite them and delete these lines.
     # This does not negatively affect the validity of the test.
-    if test_vtd_plugin_exists; then
-      reference_file=test_engine_nop_smoketest_dump_with_vtd.json
-    else
-      reference_file=test_engine_nop_smoketest_dump.json
-    fi
-    diff <(cloe_engine dump config_nop_smoketest.json |
-        sed -r -e 's#"/.*\/.*.conan/data/([^/]+)/.*"#"/.../\1/.../"#' -e '/\/(home|root)\/.*/d') \
+    reference_file=test_engine_nop_smoketest_dump.json
+    diff <(cloe-engine dump config_nop_smoketest.json |
+        sed -r -e 's#"/.*\/.*.conan/data/([^/]+)/.*"#"/.../\1/.../"#' -e '/\/(home|root|cloe_dev)\/.*/d') \
```

Comment:
Another, more general approach for replacing the absolute paths was suggested by @clsim . Can we make use of the environment variables `$HOME` and `$CONAN_USER_HOME`? That would probably cover most cases.